src/requirements/data/minors/robotics.ts

```diff
@@ -0,0 +1,101 @@
+import { CollegeOrMajorRequirement } from '../../types';
+import { includesWithSingleRequirement } from '../checkers-common';
+
+const roboticsMinorRequirements: readonly CollegeOrMajorRequirement[] = [
+  // TODO: can't handle major specific restrictions yet
+  // ME majors may not count MAE 3780 if it is used to satisfy the ME circuits requirement
+  // ECE majors may not count ECE 3400
+  {
+    name: 'Fundamentals Courses',
+    description: 'Choose three courses. ',
+    source:
+      'https://www.engineering.cornell.edu/students/undergraduate-students/curriculum/majors-and-minors/robotics-minor',
+    checker: includesWithSingleRequirement(
+      'CS 4750',
+      'MAE 4180',
+      'CS 4752',
+      'MAE 3780',
+      'ECE 3400',
+      'CS 4700',
+      'CS 4754'
+    ),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [3],
+    slotNames: ['Course'],
+  },
+  {
+    name: 'Specialization Courses',
+    description: 'Choose three courses from a category.',
+    source:
+      'https://www.engineering.cornell.edu/students/undergraduate-students/curriculum/majors-and-minors/robotics-minor',
+    fulfilledBy: 'toggleable',
+    fulfillmentOptions: {
+      Intelligence: {
+        description: 'Choose three courses. ',
+        counting: 'courses',
+        checker: includesWithSingleRequirement(
+          'CS 4790',
+          'CS 6751',
+          'MAE 6770',
+          'MAE 6790',
+          'ECE 6970',
+          'CS 4700',
+          'MAE 4180',
+          'MAE 6710'
+        ),
+        perSlotMinCount: [3],
+        slotNames: ['Courses'],
+      },
+      'Modelling, Dynamics, and Control': {
+        description: 'Choose three courses. ',
+        counting: 'courses',
+        checker: includesWithSingleRequirement(
+          'MAE 4730',
+          'MAE 4710',
+          'MAE 4780',
+          'CS 4752',
+          'MAE 6760',
+          'MAE 6770',
+          'MAE 6780'
+        ),
+        perSlotMinCount: [3],
+        slotNames: ['Courses'],
+      },
+      Perception: {
```

Comment:
Is Perception supposed to be a string like "Modelling, Dynamics, and Control"? Same for Intelligence